main/index.js

```diff
@@ -102,3 +106,26 @@ app.on('ready', function() {
     }
   });
 });
+
+function openAboutWindow() {
```

Comment:
This is not right place to declare a function, index.js is entry point for Installer application. Windows related menu is defined in browsr/bootstrap.js file.